Made partest fully support locker-based distribs

xeno-by opened the pull request:
This patch brings support for specialized and scalacheck tests,
and finally makes it possible to run partest for locker.

To do that, you need to write your own partest script: http://bit.ly/wl9HaH,
and keep another clone of your repository that would provide the classes
for running partest. That clone should then be build normally (ant build),
and be used to transplant the stuff that isn't built by locker, namely:
partest, actors, scalacheck, forkjoin, fjbg, msil and jline.
For more information take a look at my scavenger: http://bit.ly/AjMiud.

paulp commented:
This NPEs starting partest.  See https://jenkins.typesafe.com:8444/job/scala/234/console .

xeno-by committed:
Made partest fully support locker-based distribs

This patch brings support for specialized and scalacheck tests,
and finally makes it possible to run partest for locker.

To do that, you need to write your own partest script: http://bit.ly/wl9HaH,
and keep another clone of your repository that would provide the classes
for running partest. That clone should then be build normally (ant build),
and be used to transplant the stuff that isn't built by locker, namely:
partest, actors, scalacheck, forkjoin, fjbg, msil and jline.
For more information take a look at my scavenger: http://bit.ly/AjMiud.

xeno-by commented:
Sorry, should work now.
